app/Http/Controllers/Auth/ChangePasswordController.php

```diff
@@ -5,6 +5,7 @@
 use App\Http\Controllers\Controller;
 use App\Http\Requests\UpdatePasswordRequest;
 use App\Http\Requests\UpdateProfileRequest;
+use App\Model\User;
```

Comment:
Do we need to include the App\Model\User. Since you are calling auth() not User

Reply:
No, we do not. My apologies. I thought of calling it through User first, but then switched it to auth().